tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/bubber/antag_optin.tsx

```diff
@@ -0,0 +1,11 @@
+// THIS IS A BUBBERS UI FILE
+import { FeatureChoiced } from '../../base';
+import { FeatureDropdownInput } from '../../dropdowns';
+
+export const rr_opt_in_status_pref: FeatureChoiced = {
+  name: 'Be Round Removed',
+  description:
+    'Enabling any non-ghost antags \
+  (revenant, abductor contractor, etc.) will force your opt-in.',
```

Comment:
This isn't accurate to RR opt in, and needs to actually describe what the pref involves, like "This will show other players that you are okay with being round removed."

Reply:
oh yeah thanks for reminder forgot about this one